recipes/openexrpython/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "OpenEXR" %}
+{% set version = "1.3.2" %}
+
+package:
+  name: openexrpython  # clash with existing openexr C++ library
```

Comment:
usually this would be `openexr-python`

Reply:
The repo on GitHub is named without the hyphen; do you think that's worth emulating?

Reply:
Went ahead and renamed it.